pkg/app/web/src/components/delete-application-dialog.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { FC, memo, useCallback } from "react";
+import {
+  makeStyles,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Button,
+  Typography,
+  CircularProgress,
+} from "@material-ui/core";
+import { useSelector, useDispatch, shallowEqual } from "react-redux";
+import {
+  selectById,
+  Application,
+  fetchApplications,
+} from "../modules/applications";
+import {
+  clearDeletingApp,
+  deleteApplication,
+} from "../modules/delete-application";
+import { AppState } from "../modules";
+import Alert from "@material-ui/lab/Alert";
+import { AppDispatch } from "../store";
+import { red } from "@material-ui/core/colors";
+import { UI_TEXT_CANCEL, UI_TEXT_DELETE } from "../constants/ui-text";
+import { useStyles as useButtonStyles } from "../styles/button";
+import { Skeleton } from "@material-ui/lab";
+import { addToast } from "../modules/toasts";
+import { DELETE_APPLICATION_SUCCESS } from "../constants/toast-text";
+
+const useStyles = makeStyles((theme) => ({
+  applicationName: {
+    color: theme.palette.text.primary,
+    fontWeight: theme.typography.fontWeightMedium,
+  },
+  description: {
+    marginBottom: theme.spacing(2),
+  },
+  deleteButton: {
+    color: theme.palette.getContrastText(red[400]),
+    backgroundColor: red[700],
+    "&:hover": {
+      backgroundColor: red[700],
+    },
+  },
+}));
+
+const TITLE = "Delete Application";
+const ALERT_TEXT = "Are you sure you want to delete the application?";
```

Comment:
If anyone has a better phrase, please let me know.

Reply:
Looks good to me.
"Are you sure you want to DELETE the application?"

Reply:
We could you the same text as github does 🤔 
<img width="461" alt="Screen Shot 2020-12-22 at 19 08 46" src="https://user-images.githubusercontent.com/32532742/102876872-32e4dd00-4489-11eb-8fea-285fa7adff8f.png">

Reply:
👍

Reply:
The confirmation could be added/improved later.
Let's merge this pull request with the current design.

Reply:
🙆‍♀️ 🚀